include/engine/search_engine_data.hpp

```diff
@@ -75,12 +79,8 @@ template <> struct SearchEngineData<routing_algorithms::mld::Algorithm>
 
     static SearchEngineHeapPtr forward_heap_1;
     static SearchEngineHeapPtr reverse_heap_1;
-    static SearchEngineHeapPtr forward_heap_2;
```

Comment:
I have added dummy heaps for another reason and before @daniel-j-h asked me why i added them, so at the moment it is a technical debt.

Good place to re-introduce them would be an alternative search implementation PR